Livecheckables/szip.rb

```diff
@@ -1,6 +1,6 @@
 class Szip
   livecheck do
     url "https://support.hdfgroup.org/ftp/lib-external/szip/"
-    regex(%r{href=.+?v?(\d+(?:\.\d+)+)/?["']})
+    regex(%r{href=.*?v?(\d+(?:\.\d+)+)/?[ '">]}i)
```

Comment:
```suggestion
    regex(%r{href=.*?v?(\d+(?:\.\d+)+)/?["' >]}i)
```

The order that I've been using is `["' >]` and this is used in a number of livecheckables right now. There do appear to be a handful of uses of `[ '">]` (11 in total) but we'll need to bring them up to date sometime afterward.